pkgs/applications/networking/cluster/helm/wrapper.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, symlinkJoin, lib, makeWrapper
+, writeText
+}:
+with stdenv.lib;
+
+helm:
+
+let
+  wrapper = {
+    plugins ? [],
+    extraMakeWrapperArgs ? ""
+  }:
+  let
+
+  initialMakeWrapperArgs = [
+      "${helm}/bin/helm" "${placeholder "out"}/bin/helm"
+      "--argv0" "$0"
+      "--set" "HELM_PLUGINS" "${pluginsDir}"
+    ];
+    pluginsDir = concatStringsSep ":" plugins;
+  in
+  symlinkJoin {
```

Comment:
why `symlinkJoin` with a singleton `paths` and then `rm $out/bin/helm`?

Reply:
because paths copies everything (manpage etc), but makeWrapper requires no preexising `$out/bin/helm` hence the `rm` call